templates/fullstack/server/src/index.ts

```diff
@@ -34,7 +34,7 @@ async function start() {
   const apolloServer = new ApolloServer({
     typeDefs: loadSchemaFiles(join(__dirname, '/schema/')),
     resolvers: loadResolversFiles(join(__dirname, '/resolvers/')),
-    context: createKnexRuntimeContext(db, pubSub),
+    context: createKnexRuntimeContext(db as any, pubSub),
```

Comment:
Amazing change that will help to resolve knex incompatibilities. Generally knex team is breaking types with every release they do so using different versions often causing compilation issues. CC @craicoverflow